### ISSUE TITLE
Can't build delve on Windows without godeps

### ISSUE THREAD

**Issue by ignatov** [derekparker/delve#392]
Hi, I've tried to build the latest delve from the source code with the vendoring support (and without godeps) and failed:
```
[22:45:18]go install -ldflags="-X main.Build=06bba9b7a9ad0e06f4d74bf71f7667d24872a838" github.com/derekparker/delve/cmd/dlv
[22:45:19]proc\proc_windows.go:20:2: cannot find package "golang.org/x/sys/windows" in any of:
[22:45:19]	C:\BuildAgent\work\gopath\src\github.com\derekparker\delve\vendor\golang.org\x\sys\windows (vendor tree)
[22:45:19]	C:\BuildAgent\work\go\src\golang.org\x\sys\windows (from $GOROOT)
[22:45:19]	C:\BuildAgent\work\gopath\src\golang.org\x\sys\windows (from $GOPATH)
```

**Comment by derekparker** [comment]
Looks like this dependency was never actually vendored. Probably an oversight from the original patch, which makes me wonder how this is working in CI. I will submit a patch shortly to vendor it properly.

**Comment by lukehoban** [comment]
> Probably an oversight from the original patch, which makes me wonder how this is working in CI.

Yeah - I'm actually not clear how this is working in CI.  But the Appveyor CI is still using godep instead of GO15VENDOREXPERIMENT, which is likely the root of the difference.

After vendoring `golang.org/x/sys/windows` per #393, we should update appveyor.yaml with:

```diff
diff --git a/appveyor.yml b/appveyor.yml
index 5fe1ee6..a312275 100644
--- a/appveyor.yml
+++ b/appveyor.yml
@@ -14,12 +14,11 @@ install:
               &7z x -oC:\ $file > $null
       }
   - set PATH=c:\mingw64\bin;%GOPATH%\bin;%PATH%
+  - set GO15VENDOREXPERIMENT=1
   - echo %PATH%
   - echo %GOPATH%
   - go version
   - go env
-  - go get github.com/tools/godep
-  - godep restore
 cache: C:\mingw64
 build_script:
 - mingw32-make install
```

You can either do that along with #393, or I'll open a follow-up PR later once that is merged.

I can also confirm that using GO15VENDOREXPERIMENT in the CI without #393 does fail with the error @ignatov mentions, as expected.
